src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.tsx

```diff
@@ -168,10 +170,58 @@ function AttachmentPickerWithMenuItems({
             },
         };
 
-        return ReportUtils.temporary_getMoneyRequestOptions(report, policy, reportParticipantIDs ?? []).map((option) => ({
+        let list: PopoverMenuItem[] = ReportUtils.temporary_getMoneyRequestOptions(report, policy, reportParticipantIDs ?? []).map((option) => ({
             ...options[option],
         }));
-    }, [translate, report, policy, reportParticipantIDs, isDelegateAccessRestricted]);
+
+        if (canUseCombinedTrackSubmit) {
+            const trackOnlyCreateExpense = {
+                icon: getIconForAction(CONST.IOU.TYPE.CREATE),
+                text: translate('iou.createExpense'),
+                onSelected: () => selectOption(() => IOU.startMoneyRequest(CONST.IOU.TYPE.TRACK, report?.reportID ?? '-1'), true),
+            };
```

Comment:
You could just reuse the object from `options`, no?

Reply:
No, since we need to use the before methods for onSelected, but the new icon and text

Reply:
updated to reuse